Packs/Carbon_Black_Enterprise_Live_Response/Integrations/Carbonblackliveresponse/Carbonblackliveresponse.js

```diff
@@ -1061,6 +1061,7 @@ function getCommandInfo(sessionId, commandId, timeout, timeInterval, cancelOnPen
     let result;
     let curTry = 0;
     while (curTry < retries) {
+        sessionKeepaliveRequest(args.session); // Keep interactive process alive while polling for results.
```

Comment:
Can this be placed above the while loop? Since each loop will do an API call to keep the interactive process alive, and that can be costly. Or is it necessary to be called right before the **getCommandRequest** function?